examples/gallery/plot/double-y-axes.py

```diff
@@ -0,0 +1,63 @@
+"""
+Double Y-axes
+-------------
+
+The ``frame`` parameter of the plotting methods of the :class:`pygmt.Figure`
+class can control which axes should be plotted and optionally show annotations
+tick marks, and grid-lines. By default, all 4 axes are plotted, along with annotations
+and tick marks (denoted **W**, **S**, **E**, **N**). Lower case versions (**w**, **s**,
+**e**, **n**) can be used to denote to only plot the axes with tick marks. We
+can also only plot the axes without annotations and tick marks using **l**
+(left axis), **r** (right axis), **t** (top axis), **b** (bottom axis).
+When ``frame`` is used to change the frame settings, any axes
+that are not defined using one of these three options are not
+drawn.
+"""
```

Comment:
Could you add a short description of how double Y-axes graphs are plotted in GMT. For example, 

> To plot a double Y-axes graph using PyGMT, we need to plot two base maps separately. 
> The two base maps should share the same projection parameter and x-axis limits, but
> different y-axis limits.

Reply:
@seisman  done in f054e48.